src/patternfly/components/Nav/nav.scss

```diff
@@ -1,1210 +1,415 @@
-
-  // Light subnav
-  --#{$nav}--m-light__subnav__link--hover--after--BorderColor: var(--#{$pf-global}--BorderColor--dark-100);
-  --#{$nav}--m-light__subnav__link--focus--after--BorderColor: var(--#{$pf-global}--BorderColor--dark-100);
-  --#{$nav}--m-light__subnav__link--active--after--BorderColor: var(--#{$pf-global}--BorderColor--dark-100);
-  --#{$nav}--m-light__subnav__link--m-current--after--BorderColor: var(--#{$pf-global}--active-color--100);
-
-  // List
-  --#{$nav}__list--PaddingTop: var(--#{$pf-global}--spacer--sm);
-  --#{$nav}__list--PaddingBottom: var(--#{$pf-global}--spacer--sm);
-
-  // Item
-  --#{$nav}__item--MarginTop: 0;
-  --#{$nav}__item--m-current--not--m-expanded__link--BackgroundColor: var(--#{$pf-global}--BackgroundColor--dark-400);
-  --#{$nav}__link--m-current--not--m-expanded__link--after--BorderWidth: var(--#{$pf-global}--BorderWidth--xl);
-  --#{$nav}__item__item__link--PaddingRight: var(--pf-t--global--spacer--xl);
-  --#{$nav}__item__item__toggle--FontSize: var(--#{$pf-global}--FontSize--sm);
-  --#{$nav}__item__toggle-icon--Rotate: 0;
-  --#{$nav}__item--m-expanded__toggle-icon--Rotate: 90deg;
-  --#{$nav}__item--BorderRadius: var(--pf-t--global--border--radius--small);
-
-  // Item ::before borders
-  --#{$nav}__item--before--BorderColor: var(--#{$pf-global}--BackgroundColor--dark-200);
-  --#{$nav}__item--before--BorderWidth: var(--#{$pf-global}--BorderWidth--sm);
-
-  // Link
-  --#{$nav}__link--FontSize: var(--pf-t--global--font--size--body--default);
-  --#{$nav}__link--FontWeight: var(--pf-t--global--font--weight--body);
-  --#{$nav}__link--PaddingTop: var(--pf-t--global--spacer--md);
-  --#{$nav}__link--PaddingRight: var(--pf-t--global--spacer--md);
-  --#{$nav}__link--PaddingBottom: var(--pf-t--global--spacer--md);
-  --#{$nav}__link--PaddingLeft: var(--pf-t--global--spacer--md);
-  --#{$nav}__link--BorderRadius: var(--pf-t--global--border--radius--small);
-  --#{$nav}__link--xl--PaddingRight: var(--pf-t--global--spacer--lg);
-  --#{$nav}__link--xl--PaddingLeft: var(--pf-t--global--spacer--lg);
-  --#{$nav}__link--Color: var(--pf-t--global--text--color--subtle);
-  --#{$nav}__link--hover--Color: var(--pf-t--global--text--color--subtle);
-  --#{$nav}__link--focus--Color: var(--pf-t--global--text--color--subtle);
-  --#{$nav}__link--active--Color: var(--pf-t--global--text--color--subtle);
-  --#{$nav}__link--m-current--Color: var(--pf-t--global--text--color--regular);
-  --#{$nav}__link--BackgroundColor: transparent;
-  --#{$nav}__link--hover--BackgroundColor: var(--pf-t--global--background--color--primary--default);
-  --#{$nav}__link--focus--BackgroundColor: var(--pf-t--global--background--color--primary--default);
-  --#{$nav}__link--active--BackgroundColor: var(--pf-t--global--background--color--primary--default);
-  --#{$nav}__link--m-current--BackgroundColor: var(--pf-t--global--background--color--primary--default);
-  --#{$nav}__link--OutlineOffset: calc(var(--#{$pf-global}--spacer--xs) * -1);
```

Comment:
This would be worth adding back in, as right now the focus outline gets cut off in horizontal examples:

![image](https://github.com/patternfly/patternfly/assets/70952936/a6a3edc2-34cd-4bb9-8fc5-f498c6feb2e2)